krabcake/rs_hello/src/lib.rs

```diff
@@ -126,6 +137,10 @@ pub struct Context {
     normalize_output: bool,
 }
 
+// FIXME: Find a better way to pass around the current thread ID.
```

Comment:
hmm. I continue to confuse myself about host vs client, and this is another instance of that. Do you know whether the thread-id we pass into `vgPlain_get_and_pp_StackTrace` is meant to be the thread id of the client, or that of the host? I'm currently inferring that it must be the thread-id of the client, but then that sends me back into wondering, "is this actually doing what we think its doing?" 😆